qsignature/src/org/qcmg/sig/util/ComparisonUtil.java

```diff
@@ -6,9 +6,6 @@
  */
 package org.qcmg.sig.util;
 
```

Comment:
have you deleted the old file and write a new one? they look very different in the comparison window.  But I think they do the same work for both old and new files. Is the related unit test updated?

Reply:
Not sure what happened - this file hasn't really changed...